tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -17,12 +18,16 @@ import {
   fakeAddon,
   fakeI18n,
   fakeTheme,
+  fakeVersion,
   shallowUntilTarget,
 } from 'tests/unit/helpers';
 import LoadingText from 'ui/components/LoadingText';
 
 describe(__filename, () => {
-  const { store } = dispatchClientMetadata();
```

Comment:
D'oh. How did that creep in? Sigh.

I did a quick search. This seemed to be the only one.

Reply:
Are you just commenting that this was a problem before so it's good that I changed it, or are you pointing out something I need to change here?

Reply:
Nothing to change. I was just surprised that the old code was so broken. Thanks for fixing it.